### PR TITLE
Add list and create endpoints for routers

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by dubesoftware** [comment]
Looks about right and is syntactically sound. My current knowledge of Junebug internals doesn't allow for more analysis.

File: junebug/router/base.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        if self.junebug_config.replace_routers:
+            return self.junebug_config.routers
+        else:
+            routers = {}
+            routers.update(default_router_types)
+            routers.update(self.junebug_config.routers)
+            return routers
+
+    @property
+    def _worker_class_name(self):
```
Comment (KaitCrawford) [inline junebug/router/base.py]: I don't see a test for this. Am I missing it or are you planning to add it later?

Comment (rudigiesler) [reply junebug/router/base.py]: Ah, no it seems there isn't. I'll add it as a test for the `start` function.

**Comment by rudigiesler** [comment]
@KaitCrawford Changes addressed, ready for another review

**Pull request merged**